details/details.html

```diff
@@ -4,8 +4,25 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./details.css/" />
-    <title>Document</title>
+    <link rel="stylesheet" href="style/style.css" />
```

Comment:
fix the link of the style file 
```suggestion
    <link rel="stylesheet" href="./details.css" />
```

Reply:
Modified